sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -56,6 +56,8 @@ public String getArguments() {
         return arguments;
     }
 
+    public String getOriginalExamples() { return examples; }
```

Comment:
I don't know that this is worth adding just for testing?

Reply:
+1 for @srowen 's opinion.

@MaxGekk . Shall we use the following format in the test case?
```scala
val examplesField = classOf[ExpressionInfo].getDeclaredField("examples")
examplesField.setAccessible(true)
val examples = examplesField.get(e).asInstanceOf[String]
```

Reply:
Or can we use `@VisibleForTesting` at least?